frame/alliance/src/lib.rs

```diff
@@ -929,12 +976,18 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	}
 
 	/// Collect all members who have voting rights into one list.
-	fn votable_members_sorted() -> Vec<T::AccountId> {
+	fn votable_members() -> Vec<T::AccountId> {
```

Comment:
It might not be a big deal since it's not the public API of this pallet, and also it's probably my fault for not changing it in a prior PR, but "votable members" would generally mean "members who can be voted for", _not_ "members who have voting rights".

I'd make a separate trivial PR to rename this to `voting_members`.

Not at all a blocker for this PR, just mentioning it.